README.md

```diff
@@ -0,0 +1,34 @@
+# Gallerya
+
+Gallerya transforms the WordPress native post gallery into a full fledged slideshow with features like images thumbnails stripe and lightbox support.
```

Comment:
@juanlopez4691  The original readme says `like images thumbnails stripe`, but it's not clear what that is. Should we re-word this?

Reply:
If you can think of a better way to express the idea, I don't see why we shouldn't fix that 👍